bsd/models.py

```diff
@@ -43,9 +43,10 @@ def assert_valid_account(api_result):
     assert cons.findtext('is_banned') == "0"
 
 
-def find_constituents_by_state_cd(state_cd):
+def find_constituents_by_state_cd(state_cd, cons_group=None):
```

Comment:
would be good to update params doc for method too. seems like it should be a string id or a list of string ids, right?